lib/kitchen/driver/oci.rb

```diff
@@ -161,60 +175,130 @@ def net_api
         generic_api(OCI::Core::VirtualNetworkClient)
       end
 
+      def dbaas_api
+        generic_api(OCI::Database::DatabaseClient)
+      end
+
+      ##################
+      # Common methods #
+      ##################
       def launch_instance(state)
-        request = compute_instance_request(state)
+        if instance_type == 'compute'
+          launch_compute_instance(state)
+        elsif instance_type == 'dbaas'
+          launch_dbaas_instance
+        end
```

Comment:
Same comment as above, lets check for the fail through case where nothing matches.